### ISSUE TITLE
UPDATE SECTION (Handoffs & renewals) - Post-handoff ATO

### ISSUE THREAD

**Issue by annahsgraves** [18F/product-guide#21]
Note from @afeld on the **Renewals & Handoffs** section - 

"Is there anything that needs to happen in regards to the ATO? Do they need to re-complete one within their agency?"

**Comment by annahsgraves** [comment]
@afeld is this covered at all in the After You Ship/ATO guide? I wonder if it makes more sense to be there than in Handoffs & Renewals.

CC: @brethauer @theresaanna @leahbannon for awareness, since it may affect their sections.

**Comment by afeld** [comment]
> is this covered at all in the After You Ship/ATO guide?

Not exactly. We have [a general description of ATOs](https://pages.18f.gov/before-you-ship/ato/) (which [needs improvement](https://github.com/18F/before-you-ship/issues/84)), though there's only so much we can say broadly, since the ATO process/requirements differ from agency to agency.

We do have a page on [ATO delegation](https://pages.18f.gov/before-you-ship/ato/delegation/), though that would be done prior to the project being publicly launched, rather than as part of the handoff. For systems where the ATO _is_ delegated, though, I'm not sure if we retain the ATO even if another agency has taken on the development/operational responsibility.

In other words, it seems like there are two paths for Delivery projects: they do their own ATO, or they delegate their ATO us (preferred, presumably). For the former, they would need to check with their agency if anything needs to happen during the handoff. For delegated ATOs, I'm not sure if that status changes or not. If it does, then I would guess they need to do their _own_ ATO on their side.

Does that make sense?

> I wonder if it makes more sense to be there than in Handoffs & Renewals.

No strong feelings from me.

/cc @18F/ato

**Comment by annahsgraves** [comment]
Thanks @afeld. I want to make sure we get this into the right spot. Sounds like it is as you suggested since it's deeply relevant when we're handing off. I'm thinking it might be something we want to capture in both places though, since the **After You Ship/ATO** guide should be all-inclusive of ALL THE THINGS. Want to take a stab at a PR? 

@NoahKunin I know you're planning on contributing SME edits on the **Renewals & Handoffs** section. What are your thoughts on this?

**Comment by annahsgraves** [comment]
Would like to also add a reference to the [#compliance-toolkit](https://18f.slack.com/archives/compliance-toolkit) resource here if it makes sense.
